src/store/modules/network.ts

```diff
@@ -68,8 +68,8 @@ export default {
   mutations: { 
     setNetworkData(state) {
       // TODO
-      state.networkName = "Songbird";
-      state.networkCurrency = "SGB";
+      state.networkName = "Flare";
```

Comment:
"Coston Testnet" also here

Reply:
@chnejohnson I see you didn't get a notification about this comment because I forgot to tag you :)